library/network_connections.py

```diff
@@ -1607,12 +1617,16 @@ def connections(self):
     def connections_data(self):
         c = self._connections_data
         if c is None:
-            assert self.check_mode in [
+            if self.check_mode not in [
                 CheckMode.DRY_RUN,
                 CheckMode.PRE_RUN,
                 CheckMode.REAL_RUN,
-            ]
+            ]:
+                raise AssertionError(
+                    "invalid value {0} for self.check_mode".format(self.check_mode)
+                )
             c = []
+            # pylint: disable=blacklisted-name
```

Comment:
```suggestion
            # disable check for bad names:
            # pylint: disable=C0102
```

The name for the C0102 check does not match our conscious language initiative, therefore using the code with an explanation instead of the name seems to be better here. What is the bad variable name here? `_`? Is there a better pattern to do something x times than a `for _ in range()`? Maybe we can get rid of the warning completelely.

Reply:
I rewrote the code to avoid this issue - check it out